source/_integrations/buienradar.markdown

```diff
@@ -6,125 +6,108 @@ ha_category:
-  required: false
-  type: float
-longitude:
-  description: Longitude to use for selection of data source location. Longitude and latitude will be taken from Home Assistant configuration but can be overridden/changed in this integration to select a different location for Buienradar.
-  required: false
-  type: float
-forecast:
-  description: "`true` to add a temperature forecast, `false` to suppress it."
-  required: false
-  type: boolean
-  default: true
-{% endconfiguration %}
-
-### Full configuration
-
-A full configuration example:
-
-```yaml
-# Example configuration.yaml entry
-weather:
-  - platform: buienradar
-    name: "volkel"
-    # Force 'Meetstation Volkel' to be used:
-    latitude: 51.65
-    longitude: 5.70
-    forecast: true
-```
-
-
-<div class='note'>
-
-This platform is an alternative to the [`buienradar`](/integrations/sensor.buienradar/) sensor.
-The weather platform is easier to configure but less customizable.
-
-</div>
+{% include integrations/config_flow.md %}
 
 ## Camera
 
-The `buienradar` camera platform uses [buienradar.nl](https://buienradar.nl/) as a source for the last rain radar map. The overview image of the whole of the Netherlands is loaded and shown as a camera in Home Assistant.
+The `buienradar` camera platform uses [buienradar.nl](https://buienradar.nl/) as a source for the last rain radar map. The overview image of the whole of the Netherlands is loaded and shown as a camera in Home Assistant. The Netherlands is the default country and can be changed to Belgium (see [Setting options](#setting-options)).
 
 Internally this component uses the radar map image as [documented](https://www.buienradar.nl/overbuienradar/gratis-weerdata) on buienradar.nl.
 The downloaded image is cached to prevent Home Assistant from making a new request to buienradar.nl multiple times a minute when Home Assistant checks for new stills from the camera.
 
-To add `buienradar` camera to Home Assistant, add the following section to your
-`configuration.yaml` file:
-```yaml
-# Example configuration.yaml entry
-camera:
-  - platform: buienradar
-```
+The camera entity is added disabled by default and should first be enabled before it starts reading the camera images.
+
+## Sensor
+
+The `buienradar` sensor platform will setup separate sensor entities with more detailed weather data. The selected weather station will provide all weather data, with the exception of the forecasted precipitation. The forecasted precipitation data will be retrieved from buienradar using your actual gps-location (and not the location of the nearest weather station). The sensor entities are disabled by default and should be enabled before they will be updated with data.
+
+The following entities will be created:
+
+- `stationname`: The name of the selected meteo-station.
```

Comment:
We use code fencing for code & file names.
This might make people think this is the name of the entity (which is not guaranteed and can be customized).

Instead, just make a list of descriptions, or, replace the code fence with `**Station name**`